dataprofiler/labelers/char_load_tf_model.py

```diff
@@ -237,7 +237,8 @@ def _construct_model(self) -> None:
         model_loc = self._parameters["model_path"]
 
         self._model: tf.keras.Model = tf.keras.models.load_model(model_loc)
-        softmax_output_layer_name = self._model.outputs[0].name.split("/")[0]
+        self._model = tf.keras.Model(self._model.inputs, self._model.outputs)
```

Comment:
Required for the function to have `output_names`. Sequential models do not have that output.